test/unit/io_managers_test.py

```diff
@@ -360,12 +360,12 @@ def test_error_when_reading_view_without_metadata(fake_pudl_sqlite_io_manager_fi
 
 
 def test_ferc_xbrl_sqlite_io_manager_dedupes(mocker, tmp_path):
-    db_path = tmp_path / "test_db.sqlite"
+    db_path = tmp_path / "ferc1_xbrl.sqlite"
```

Comment:
i think this change is unnecessary and a little bad. this test actually adds the little test table into the database (via the line `df.to_sql("test_table_instant", conn)`). So whenever this unit test is run a db table is being added to the main `ferc1_xbrl` db.

Also, I did a quick poke at it and i don't think the way you've currently changed the io_managers requires this name change.

Reply:
Unfortunately the test fails if the sqlite isn't named the same as `db_name`. I'll look into how we can work around this.

Reply:
i gave this a quick test locally and it looks like if you rename all of the instances in this test it works fine.

Reply:
Can you push that code? I keep getting a `ValueError`